src/components/radioFieldSet.js

```diff
@@ -0,0 +1,94 @@
+import PropTypes from 'prop-types';
+import {
+  Button, Fieldset,
+  MultiChoice, Radio
+} from 'govuk-react'
+import { Field, Form } from 'react-final-form';
+import {Component} from 'react';
+
+class RadioFieldSet extends Component {
+  constructor(props) {
+    super(props);
+    this.name = this.props.props.name;
+    this.label = this.props.props.label;
+    this.hint = this.props.props.hint;
+    this.options = this.props.props.options;
+    this.title = this.props.props.title;
+    this.onSubmit = this.props.props.onSubmit;
+    this.beforeButton = this.props.props.beforeButton;
+  }
+
+  radioGroup = ({label, hint, options, input, meta}) => (
+    <div>
+      <MultiChoice label={label} hint={hint} meta={meta}>
+        {options.map(o => (
+          <Radio
+            key={o.value}
+            {...input}
+            value={o.value}
+            checked={o.checked}
+          >
+            {o.title}
+          </Radio>
+        ))}
+      </MultiChoice>
+    </div>
+  );
+
+  formSubmit = (params)=>{
```

Comment:
Nitpick: inconsistent spacing around arrow operator here and other files too - guessing we need to improve our linting rules(?)
```suggestion
  formSubmit = (params) => {
```